java/src/test/java/com/google/appengine/tools/pipeline/PipelinesErrorHandlingTest.java

```diff
@@ -415,7 +418,7 @@ public void testChildCancellation() throws Exception {
    */
   public void testGrandchildCancellation() throws Exception {
     String pipelineId = service.startNewPipeline(new TestGrandchildCancellationJob());
-    waitUntilTaskQueueIsEmpty();
+    waitUntilTaskQueueIsEmpty(taskQueue);
```

Comment:
Not critical. I understand why you did this, but IMHO makes the tests less readable, I was wondering where this `taskQueue` come from... If it's the only case, we could keep it, or keep extending the base test class.